### ISSUE TITLE
Let's make the test-cases completely independent

### ISSUE THREAD

**Issue by praiskup** [sclorg/postgresql-container#140]
So we can selectively run particular tests.

**Comment by praiskup** [comment]
And run the tests in parallel, ideally through `make` which is proven to handle parallelism correctly.

**Comment by pkubatrh** [comment]
I will try taking a look at this.
Running test cases in parallel will be problematic right now with the master_restart test relying on a specific IP of the master to synchronize properly (which might get 'stolen' by another running test case) so that will definitely need to be fixed before making the change.

**Comment by praiskup** [comment]
Isn't it possible to start docker containers so they see only explicit subset of other containers?

Let's make the test-cases as much independent as possible first.  The parallel approach is something which deserves another issue.

**Comment by pkubatrh** [comment]
>Isn't it possible to start docker containers so they see only explicit subset of other containers?

Right, creating a docker network for a single test case should resolve the issue. Will try it out later and focus on test case independency for now.
